rskj-core/src/main/java/org/ethereum/vm/trace/OpActions.java

```diff
@@ -135,7 +135,7 @@ public Action addMemoryExtend(long delta) {
     public Action addMemoryWrite(int address, byte[] data, int size) {
         return addAction(memory, Action.Name.WRITE)
                 .addParam("address", address)
-                .addParam("data", toHexString(data).substring(0, size));
+                .addParam("data", toHexStringOrEmpty(data).substring(0, size));
```

Comment:
My main concern with this PR is the replacement of toHexString with toHexStringOrEmpty, like for example in this line... In many cases it is not replaced, but in some cases it is replaced, even though it clearly cannot be null. Why is that?